base_classes/NXdata.nxdl.xml

```diff
@@ -83,25 +83,28 @@
 		<doc>
 			.. index:: plotting
 			
-			String array that defines the independent data fields used in 
-			the default plot for all of the dimensions of the *signal* field
-			(the *signal* field is the field in this group that is named by
-			the ``signal`` attribute of this group). 
-			One entry is provided for every dimension in the *signal* field.
+			Array of strings holding the :ref:`names &lt;validItemName>` of
+			the independent data fields used in the default plot for all of
+			the dimensions of the :ref:`signal &lt;/NXdata@signal-attribute&gt;`
+			or :ref:`auxiliary signals &lt;/NXdata@auxiliary_signals-attribute&gt;`.
+			
+			One name is provided for every dimension in the *signal* or *auxiliary signal* fields.
 			
-			The field(s) named as values (known as "axes") of this attribute 
-			*must* exist. An axis slice is specified using a field named 
-			``AXISNAME_indices`` as described below (where the text shown here
-			as ``AXISNAME`` is to be replaced by the actual field name).
+			The *axes* values are the names of fields or links that *must* exist and be direct
+			children of this NXdata group.
+
+			An axis slice is specified using a field named ``AXISNAME_indices``
```

Comment:
Do we define what we need by an axis slice? Simpler to say "An axis's dimensional mapping to the signal data field...".

Reply:
So if I understand correctly: for each signal dimension you have exactly one axis, but each axis can be used for one or more dimensions?

Reply:
Actually I don't understand why `AXISNAME_indices` exists. The `axes` attribute is enough.

Reply:
Anyway, `AXISNAME_indices` is not related to this issue. I'' make another issue for this.

Reply:
If your axis dataset is multi-dimensional, then you need to index into that from the position in your signal dataset. I.e. `signal` has shape `(I,J,K,L)` for `axes=[pressure,temperature,.,.]`, where `pressure` has shape `(I,K,L)` and `temperature` has shape `(J,)`, then `pressure_indices=[0,2,3]` and `temperature_indices=[1]`

Reply:
https://github.com/nexusformat/definitions/issues/1212

Reply:
Ok I had absolutely no idea that was a possibility. The docs definitely need improving but not in this PR.